templates/default/snapraid_runner.conf.erb

```diff
@@ -1,14 +1,19 @@
+<%
+path_separator = node['platform'] == 'windows' ? '\\' : '/'
+executable_extension = node['platform'] == 'windows' ? '.exe' : ''
+executable = node['snapraid']['install_directory'] + path_separator + 'snapraid' + executable_extension
+%>
 [snapraid]
 ; path to the snapraid executable (e.g. /bin/snapraid)
-executable = /usr/local/bin/snapraid
+executable = <%=executable%>
```

Comment:
I would move all of these into attribs. That way you dont need any logic inside the template.

```ruby
default['snapraid']['executable'] = node['snapraid']['install_directory'] + File:SEPERATOR + 'snapraid'
```

Though, I seem to remember issues with attributes based off of other attributes.